stories/CodeShape.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import CodeShape, { generateLineShapes } from '../src/components/CodeShape';
+
+const exampleCode = `\
+browser.runtime.onMessage.addListener(
```

Comment:
This looks like the longest line but it is only 39 chars. Could we have a story with a line that has more chars than the `maxLineLength`?

Reply:
If you keep scrolling in *all variants* there is one already:

<img width="462" alt="Screenshot 2019-04-12 10 32 57" src="https://user-images.githubusercontent.com/55398/56049145-71b31d80-5d0e-11e9-8b54-25cd067e9127.png">

Reply:
ah oops, I saw it actually. Sorry.